pkg/nacler/keyers/tpmkeyer/tpmkeyer.go

```diff
@@ -0,0 +1,142 @@
+package tpmkeyer
```

Comment:
not convinced on package name .... it's a bit redundant `pkg/nacler/keyers/tpmkeyer` ... but `pkg/nacler/keyers/tpm` felt weird using in code, for example calling `tpm.New()`

Reply:
I think the alternative, would be to call it `pkg/nacler/keyers/tpm`, and if places needed to import both, they'd tpmkeyer "pkg/nacler/keyers/tpm"`.

But this seemed fine to me

Reply:
renamed package to just tpm